src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -47,7 +47,7 @@ static void ReplaceSelectedWaterPalette(const ObjectRepositoryItem* item);
 
 /**
  * Master objects are objects that are not
- * optional / required dependants of an
+ * optional / required dependents of an
```

Comment:
Why did you change this?

Reply:
isn't that just a difference between British and American English?

Reply:
> isn't that just a difference between British and American English?

Not necessarily. *-ant* is for the noun, *-ent* is for the adjective.

Reply:
visual studio says dependants is not a word. But I changed it before visual studio even told me to.